packages/vite/client.d.ts

```diff
@@ -247,6 +247,11 @@ declare module '*?inline' {
   export default src
 }
 
+declare module '*?no-inline' {
+  const src: string
+  export default src
+}
+
```

Comment:
```suggestion

declare module '*?url&inline' {
  const src: string
  export default src
}

```
Let's add this one as well.